.gitignore

```diff
@@ -14,3 +14,76 @@ vendor/
 .vscode
```

Comment:
How should it be trimmed down? Just remove the '.vscode'?

Reply:
I prefer no gitignore per sample, can you add to the root fabric-samples gititgnore instead? Then it will be inherited for every sample. I believe every sample should be able to live by the same gitignore rules.

Reply:
This is the root .gitignore. Originally I copy pasted the chaincode .gitignore into the root .gitignore. Then, per Dereck's suggestion, I trimmed it down to only add node_modules/ to the root .gitignore